packages/database/src/schema.prisma

```diff
@@ -375,6 +375,7 @@ model AppealCase {
   appellantGreenBelt Boolean?
 
   siteAreaSquareMetres Decimal?
+  siteAreaUnits        String?
```

Comment:
do we need this on appeal case for now? Should probably wait until defined in data model to make things easier going forward

Reply:
Fair point, removed it for now